resolver-dns-native-macos/src/main/java/io/netty/resolver/dns/macos/MacOSDnsServerAddressStreamProvider.java

```diff
@@ -120,6 +130,7 @@ public MacOSDnsServerAddressStreamProvider() {
         if (resolvers == null || resolvers.length == 0) {
             return Collections.emptyMap();
         }
+        Arrays.sort(resolvers, Collections.reverseOrder(RESOLVER_COMPARATOR));
```

Comment:
Store the instance that is returned by reverseOrder in the static field directly and reuse ?

Reply:
yep ... whatever fits you best.

Reply:
Done.